### DISCUSSION TITLE
Opensource on cv

### DISCUSSION THREAD

**Discussion by AllanRegush** [EddieHubCommunity/support#263] [Q&A] [answered]
What the is the best way to highlight open source work on your cv? I’ve never really found a good answer.

**Comment by eddiejaoude** [comment, marked as answer]
Great questions 👍 .

I believe a must is a link to ones GitHub profile and then selected ideally 3 different projects and have the following information for each:
- title
- description
- why including this project, what is great about it? Testing, Coding, Documentation etc

I will have a think and update this later on with more ideas 🤓

**Reply by Agrover112** [reply]
So this should be besides our personal projects?Cause I have many which are mix of both

**Reply by eddiejaoude** [reply]
Personal projects should be open source too for maximum benefit 🤓

**Reply by Agrover112** [reply]
Yup they are Open Source too . But I wanted to ask like some projects are done individually and other where I have contributed /worked with others on the projects .

**Reply by eddiejaoude** [reply]
Oh I see, good question 👍 . Yes I would have a mix of both, as both show different types of skills.

**Reply by BOLT04** [reply]
I would add the following:
- GitHub link for each project. Like actually using the GitHub icon and making it a link on the PDF, or making the title a link 😃 
- Maybe showcasing some of your PRs is a good idea 🤔. I'm thinking this is good because it shows how you interact and respond to feedback (PR comments), or how you give feedback. Basically showcasing with proper examples how you communicate, not just how you write code 👍

**Reply by eddiejaoude** [reply]
I like the idea of showing a PR and the communication / collaboration side 💪 - saves them hunting around for it and finding a not so great one
